src/main/java/com/conveyal/datatools/manager/jobs/DeployJob.java

```diff
@@ -788,7 +795,106 @@ private String getRouterId() {
      * startup.
      */
     private String constructUserData(boolean graphAlreadyBuilt) {
-        // Prefix/name of JAR file (WITHOUT .jar)
+        if (deployment.r5) {
+            status.fail("Deployments with r5 are not supported at this time");
+            return null;
+        }
+        String jarName = getJarName();
+        String s3JarUrl = getS3JarUrl(jarName);
+        if (!s3JarUrlIsValid(s3JarUrl)) {
+            return null;
+        }
+        String webDir = "/usr/share/nginx/client";
+        // create otp-runner config file
+        OtpRunnerManifest manifest = new OtpRunnerManifest();
```

Comment:
Suggestion:
```java
OtpRunnerManifest manifest = buildOtpRunnerManifest(graphAlreadyBuilt)
```

Reply:
I'd also have to pass a few other variables that are only contained in this method (`jarName`, `s3JarUrl` and `webDir`). I don't think it's worth it.

Reply:
That's probably ok but makes harder to segment the steps of this method.